be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -59,8 +59,7 @@ Status ColumnChunkReader::init(int chunk_size) {
 }
 
 Status ColumnChunkReader::load_header() {
-    RETURN_IF_ERROR(_parse_page_header());
-    return Status::OK();
+    return _parse_page_header();
```

Comment:
if it's just internal method, you can remove it, just call `_parse_page_header`.

Reply:
`_parse_page_header()` was used by `load_header()` and `_try_load_dictionary()` simultaneously, so maybe this way is more clear.